appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -520,6 +520,13 @@ private boolean handleQueryString() {
 
     if (templateLoadingFlag) {  // We are loading a template, open it instead
                                 // of the last project
+
+      //check to see what kind of file is in url, binary (*.aia) or base64(*.apk)
```

Comment:
base64 has .asc as an ending. .apk is for compiled apps, which should never take this code path. Probably check against ".aia" rather than just "aia".